README.md

```diff
@@ -108,12 +108,11 @@ librosa (0.x.x, /path/to/librosa)
 
 `librosa` uses `soundfile` and `audioread` to load audio files.
 
-📝 Note that `soundfile` does not currently support MP3, which will cause librosa to
-fall back on the `audioread` library.
+📝 Note that older releases of `soundfile` (prior to 0.11) do not support MP3, which will cause librosa to fall back on the `audioread` library.
```

Comment:
For MP3 decoding, librosa depends on `soundfile` v0.11 or later if available. For earlier versions, librosa falls back on `audioread`.

Reply:
Please be aware that `soundfile` relies on `libsndfile` 1.1 for MP3 decoding. Most Linux distributions do not yet ship `libsndfile` 1.1. You can `soundfile.check_format('MP3')` to see whether MP3 decoding is available.